test/e2e/multi-disk_test.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/vmware/govmomi/vim25/types"
+	"k8s.io/utils/ptr"
+	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
+	"sigs.k8s.io/cluster-api/test/framework"
+	. "sigs.k8s.io/cluster-api/test/framework/ginkgoextensions"
+)
+
+type DiskSpecInput struct {
+	InfraClients
+	Global      GlobalInput
+	SpecName    string
+	Namespace   string
+	ClusterName string
+}
```

Comment:
can it be local with non exported fields?
```suggestion
type diskSpecInput struct {
	InfraClients
	global      GlobalInput
	specName    string
	namespace   string
	clusterName string
}
```

Reply:
Sure.  I can make those changes.